Projects/Home/Sources/ClaimStatusBar.swift

```diff
@@ -0,0 +1,56 @@
+import SwiftUI
+import hCore
+import hCoreUI
+import hGraphQL
+
+struct ClaimStatusBar: View {
+    let status: Claim.ClaimStatus
+    @State var currentStatus: Claim.ClaimStatus
+
+    @hColorBuilder var barColor: some hColor {
+        switch (status, currentStatus) {
+        case (.reopened, .reopened):
+            hTintColor.orangeOne
+        case (_, .closed):
+            hTintColor.lavenderOne
+        case (let x, let y) where x == y:
```

Comment:
What does this mean haha

Reply:
a case where the two values are equal

Reply:
Call it lhs, rhs, maybe instead? its a bit clearer, x and y makes me think about positioning 😂

Reply:
haha oh true.